Gemfile

```diff
@@ -2,7 +2,7 @@
 
 source "https://rubygems.org"
 
-ruby "~> 2.6.0"
+ruby "> 2.6.0"
```

Comment:
@pleary is this going to cause any problems for us in production?

Reply:
I'll test it out on staging too. Ruby versions are fixed on our servers and to change them we'll need reinstall Passenger and recompile nginx. We also specify 2.6 in the CI.yml, so that's what CI will use. I think ideally we'd keep `~> 2.6.0` so developers don't use features only available or changed in 2.7 which don't work in CI or on our servers. I'm happy to make a push to upgrade to 2.7 in production sooner since it seems like that's a requirement for the ARM processors on M1 Macs. So I'll test out if this will break this one deploy, and it sounds like it won't, but I think I'd prefer moving this to something like an `M1-requirements` branch or something until we update production to 2.7, and make it `ruby "~> 2.7.0"` so we keep a fixed major version of Ruby